src/main/webapp/app/overview/course-exercises/course-exercise-row.component.ts

```diff
@@ -157,6 +168,10 @@ export class CourseExerciseRowComponent implements OnInit, OnDestroy {
         return participation.results && participation.results.length > 0;
     }
 
+    isExerciseInDueDate(exercise: Exercise): boolean {
```

Comment:
- Bad function name, e.g. `hasExerciseDueDatePassed`
- Should be a util.
- No doc, e.g. the fact that no dueDate evaluates to true